upgrades/3.x/3.12.0/mongodb/api-keys-migration.js

```diff
@@ -0,0 +1,15 @@
+print('Add \'key\' and \'api\' columns in \'keys\' table');
```

Comment:
You can use the `Template literals` to avoid having to escape

Reply:
good to know, i changed it